service/src/main/java/com/chabomakers/nico/DevCorsInterceptor.java

```diff
@@ -17,8 +17,10 @@
   @Override
   protected void afterAfter(Request request, Response response) {
     if (request.host().startsWith("localhost:")) {
-      response.header("Access-Control-Allow-Origin", "http://localhost:3000");
-      response.header("Access-Control-Allow-Methods", "*");
```

Comment:
This wasn't necessary. But by making this as permissive as possible we may shortcut some cors issues in the future